rivers/yahoo-finance-prices/parser.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('lodash'),
+    moment = require('moment-timezone');
+
+module.exports = function(config, body, url, temporalDataCallback, metaDataCallback) {
+    var data = JSON.parse(body);
+
+    // This is important.
+    moment.tz.setDefault("UTC");
+
+    var dtarray = data.query.created.split('T'),
+        dateString = dtarray.shift(),
+        timeString = dtarray.shift().split('Z').shift(),
+        date = moment(dateString + ' ' + timeString, 'YYYY-MM-DD HH:mm:ss'),
+        timestamp = date.unix();
+
+    
+    var res = data.query.results.quote,
+        symbol = res.symbol,
+        fieldValues;
+
```

Comment:
I would either combine these two groups of var declarations into one, or else put each on its own line like:

```javascript
var a = 'a';
var b = 'b';
```